inc/3rd-party/hosting/wpengine.php

```diff
@@ -1,147 +0,0 @@
-<?php
```

Comment:
All functions in this file should go in `inc/deprecated/3.6.php`, to mark them as deprecated and prevent potential fatal error on update